src/Telemetry/Opt_In_Subscriber.php

```diff
@@ -62,7 +62,7 @@ public function set_optin_status() {
 		}
 
 		// Don't show the opt-in modal again.
-		update_option( $this->container->get( Opt_In_Status::class )->get_show_optin_option_name(), "0" );
+		update_option( $this->container->get( Opt_In_Template::class )->get_option_name(), "0" );
```

Comment:
Weird, we have this in two different places 🤔 I agree with using the `Opt_In_Template` method. Do you mind removing the one in `Opt_In_Status`?

https://github.com/stellarwp/telemetry/blob/3a1de41dbec811dcc66a324a4503e4c557facd6f/src/Telemetry/Opt_In_Status.php#L241-L257

https://github.com/stellarwp/telemetry/blob/3a1de41dbec811dcc66a324a4503e4c557facd6f/src/Telemetry/Opt_In_Template.php#L87-L103